### ISSUE TITLE
Consider renaming/rebranding the platform from the users' perspective

### ISSUE THREAD

**Issue by rebeccawilliams** [18F/micropurchase#235]
Currently "Micropurchase" doesn't explain very clearly what users are to do on the site, i.e. to my knowledge users do not make micro-purchases on this site. Rather It explains what the government is doing internally to make an acquisition of a pull request possible. 

Alternatives that might convey the nature of the platform better would be:
* contribute.18f.gov
* codefor.18f.gov 
* helpwanted.18f.gov 
* bid.18f.gov, microbid.18f.gov
* auction.18f.gov, opensource.18f.gov/auction 
etc

Internally the process could still be called micro-purchasing, but I suspect a platform that was named from the users' POV would be more approachable and also help (to some extent) explain what might be a new or confusing process to the public.

**Comment by rebeccawilliams** [comment]
FWIW, if this tool were to be redeployed for [other agencies (or governments)](govcode.org), codefor.[agency].gov would be very consistent and approachable IMO, though I suppose that holds up for most of the options. 

"codewith" code be substituted in for "codefor" as well, see: buildwith.org

**Comment by andrewmaier** [comment]
Really like this idea, @rebeccawilliams. We're discussing it internally :)

**Comment by 47ronin** [comment]
Wow, THIS. Now that you've articulated the true meaning of what the micro purchase program is, it makes sense. I have always assumed that it indeed involved small contract transactions for public sector projects but I didn't realize that it had a community contribution model also. "CodeFor" or "CodeWith" are very appropriate and appeal to civic-minded talent looking to engage with government. Great suggestion, @rebeccawilliams.

**Comment by andrewmaier** [comment]
For more context, we've got a rudimentary about page here: https://pages.18f.gov/micropurchase/about/

**Comment by tBaxter** [comment]
FWIW, I think "microbid" sounds much more accurate than "micropurchase"

**Comment by kangman** [comment]
µbid?

**Comment by krues8dr** [comment]
Speaking as a developer, `micro`-anything doesn't have any meaning in my world. "helpwanted" and "contribute" resonate strongly for me. I love "codewith" but it's not very readable in a uri.

**Comment by saastree** [comment]
1. MBO - Micro Business Opportunities, similar to FBO. i.e. mbo.18f.gov
2. codewithus.18f.gov

**Comment by openfly** [comment]
I'd be partial to something with a faux 50s sci-fi theme.  Like zorpax.18f.gov.  We live in the future, we should start acting like it.  More silver fabric leotards less faxes.

**Comment by stvnrlly** [comment]
I agree with @krues8dr, and particularly lean towards `helpwanted`. I'd also recommend avoiding `codewith` and `codefor` as they unnecessarily narrow the range of opportunities that people expect to find on the site.

**Comment by patrickmj** [comment]
From the link @andrewmaier provided above, this sounds similar to what you might find on challenge.gov, though with initial vetting, rather than open-ended submissions?

My first thought was something like `codewith` or `codefor`, but can't deny @stvnrlly 's concerns. 

So, how about `solvewith.18f.gov`? That'd expand beyond the narrow range implied by 'code', and put forward a positive tone that picks up on the spirit in challenge.gov.

**Comment by krues8dr** [comment]
How about just `with.18f.gov`?

**Comment by ungoldman** [comment]
+1 for `bid.18f.gov`

- short and sweet
- purpose is clear
- free of rhetorical baggage

**Comment by langford** [comment]
citizenupgrade.18f.gov

**Comment by andycochran** [comment]
👍 `bid.18f.gov` makes sense, as the term "bid" is already sprinkled throughout the copy written for the user.

**Comment by billy22g** [comment]
My concern with bid.18f.gov is that it doesn't to me convey the sort of impact that people can have. When I've talked to my dev and design friends about micropurchase, the idea of bidding on a government contract is not at all appealing to them. What's appealing is the opportunity to get involved in their community and help out on a project that's low barrier to entry, small in scope, and well-defined. I think it makes sense to pick a name that makes people feel like they're doing civic engagement, and I'm not sure "bid" conveys that. I also realize (and apologize) that I'm not offering an alternative, but rather just a reaction.

**Comment by horstm22** [comment]
gig(s).18f.gov... also short, fewer eBay connotations for those outside gov.

**Comment by zceline** [comment]
if a program goal is to get internal partner agencies to try open source and at the same time engaging new contributors, perhaps more broadly understood terms rather than specific (gov, opensource, technical) lexicon. e.g. microprojects.18f.gov or @horstm22 's gigs.18f.gov ...

**Comment by neilmartis** [comment]
@rebeccawilliams @andrewmaier @mtorres253 sprints.18f.gov as we are running mini sprints in agile

**Comment by neilmartis** [comment]
buildwithus.18f.gsa.gov

**Comment by neilmartis** [comment]
mvp.18f.gsa.gov

**Comment by neilmartis** [comment]
codewithus.gsa.gov

**Comment by eaphilly** [comment]
opengig.18f.gov.  Emphasize the 'open'.  This is the key selling point to the community. The word 'gig' is clear description of the nature of the work and well understood concept. It also doesn't pigeon hole the system to software deliables only. It could also be used as a sub domain for similar programs across a number of .gov domains.

**Comment by neilmartis** [comment]
hack4gov.18f.gov

**Comment by rebeccawilliams** [comment]
I'm glad this thread has so much activity! 

FWIW, I think 18F will have to do some deep thinking about what the future goals of this product will be when doing any sort of re-brand. 

re: @stvnrlly's comment:

> I agree with @krues8dr, and particularly lean towards helpwanted. I'd also recommend avoiding codewith and codefor as they unnecessarily narrow the range of opportunities that people expect to find on the site.

I personally think this is too broad a goal for the product as it is currently maintained all through GitHub. If you want to attract folks that are deterred by the word code (a subpopulation of non-software developers), I'd recommend not hosting this product exclusively on GitHub. If you want to pay folks for open source contributions to government tech products -- I think the current set up and the word "code" work. 

This is not my product, but my future vision for it would definitely be to see other agencies adopt this tool/method. The evolution of [sourcecode.cio.gov](sourcecode.cio.gov) and [govcode.org](govcode.org) are certainly what I had in mind when I made this suggestion:

> FWIW, if this tool were to be redeployed for other agencies (or governments), codefor.[agency].gov would be very consistent and approachable IMO, though I suppose that holds up for most of the options.

If the US Federal Government has an open source policy and 18F and other agencies have a codefor.[agency].gov mechanism/tool up and running, well, that's a world I want to live in. 🇺🇸

**Comment by neilmartis** [comment]
@rebeccawilliams Great Suggestion !

**Comment by awfrancisco** [comment]
From Twitter: marketplace

https://twitter.com/gladefrancis/status/758881821434052611

**Comment by mtorres253** [comment]
Met today. We want to consider micro-purchase along with other acquisition products. Will pull out of progress for now

**Comment by andrewmaier** [comment]
Thanks for the input, everyone! We're going to take these into account during a future branding exercise. That'll likely happen sometime in September.

**Comment by nicoleslaw** [comment]
The funding mechanism should be secondary to why it matters. I would start by better defining what this platform/app/thing does for people.

If I’m understanding correctly, we’re making it easier for members of the public to find ways to chip in and get paid to do it. That is a rough summary—but consider leading with something like that over saying it’s a reverse auction house.

Have we thought about using a domain that isn’t so closely tied to 18F? I could see something like contribute.gov or helpwanted.gov working in that case. Otherwise, it sounds like you’re working with/for 18F and that is pretty limiting.

**Comment by mtorres253** [comment]
Thanks all for the wonderful suggestions! We'll be taking a look at all the acquisitions products, including this one, so that we can come up with something that better expresses the value this platform provides (to vendors and agency customers) in the larger context of acquisitions.
